changelogs/unreleased/66646.yml

```diff
@@ -0,0 +1,3 @@
+---
+title: "PurchaseOrder / SaleOrder : Manage analytic accounts per axis"
```

Comment:
Was there no changelog for when sale order where concerned ? This ticket is only about purchase orders. Also please add a space before `Order`

Reply:
Resolved